ecosystem/sep-0042.md

```diff
@@ -0,0 +1,218 @@
+diverging from standards existing for other blockchain, like Ethereum’s TokenLists.
+
+This standard aims to find the balance between providing relevant asset information and minimizing transferred
+data size, which is especially important in case of mobile browsers. The decision to allow publication of more than one
+SAL in `stellar.toml` aims to provide means for lists segmentation by categories, as well as dealing with
+size limitations.
+
+Additional asset information can be fetched from the `stellar.toml` file hosted on the `domain` specified in the asset
+description. The `comment` field facilitates the transparent sharing of supplementary information regarding assets
+communicated by the list provider, enhancing user awareness and engagement.
+
+The decision to allow hosting icons on both HTTPS and IPFS protocols contributes to higher hosting versatility and fault
+tolerance for asset logos.
+
+List versioning allows downstream applications to refresh asset information obtained from the list only when it changes.
+
+Users can submit their feedback, report bad actors or request addition of new assets by contacting the provider
+via the communication channel specified in the `feedback` field.
+
+General interface recommendations laid out in "Composability" and "Consumer UX" sections intended to at least partially
+unify user experience of asset lists across ecosystem wallets following best practices gathered from prior experience.
+
+## Security Concerns
+
+Allowing users to unrestrictedly connect arbitrary asset lists in the application interface is potentially
+dangerous because bad actors can carry on attacks to spoof legitimate assets by mimicking asset name and logo.
+Therefore, application developer should either depend only on several trustworthy providers or utilize
+the community-maintained repository of curated asset lists.
+
+Displaying the `assets.comment` contents in the interface might confuse end-users and, depending on the context,
+can be regarded as endorsements. So it's up to the application developers whether to show these comments to end users.
+
+## Implementation
+
+- Community-managed Github repository for the [Asset Lists Catalogue]( https://github.com/stellar-asset-lists/index).  
+  It contains references to asset lists created by ecosystem providers. Client applications can use it to discover,
+  fetch available lists, and allow their users to select trusted providers in the interface. Web UI for this catalogue
+  is available [here](https://stellar.expert/asset-lists).
+- Basic [JS library](https://www.npmjs.com/package/@stellar-asset-lists/sdk) that simplifies integration of asset lists
+  into web applications.
+- [StellarExpert Top50](https://api.stellar.expert/explorer/public/asset-list/top50) asset list for Stellar Pubnet.
+- [StellarExpert Testnet Top50](https://api.stellar.expert/explorer/testnet/asset-list/top50) asset list for Stellar
+  Testnet.
+
```

Comment:
Can we add a changelog at the bottom, from the [sep-template.md](https://github.com/stellar/stellar-protocol/master/sep-template.md)?
```suggestion
## Changelog

- `v0.2.0`: Initial draft. [#1440](https://github.com/stellar/stellar-protocol/pull/1440)
```